### PR TITLE
Initial script for posting papers

### PR DESCRIPTION
**Pull request by celestemartinez**
-Changed submission invitation.reply['signatures'] to conf_id
-Changed submission invitation.reply['readers'] to everyone
-Changed submission invitation.reply['writers'] to conf_id and PCs

-Main file does not contain author emails, so I create a dictionary with key (name, title) and value email to match author names to emails from a second file

### CODE REVIEW

File: venues/L4DC/2020/Conference/python/upload-submissions.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            invitation=conference_id+'/-/Submission',
+            signatures=[conference_id],
+            writers=[conference_id, conference_id+'/Program_Chairs'],
+            readers=['everyone'],
+            content={
+                'title': row[1],
+                'authors': authors,
+                'authorids': authorids,
+                'abstract': row[3],
+                'venue': 'L4DC 2020',
```
Comment (melisabok) [inline venues/L4DC/2020/Conference/python/upload-submissions.py:70]: I was going to suggest that! :)

Comment (melisabok) [reply venues/L4DC/2020/Conference/python/upload-submissions.py:70]: Should we put a venueid field too?

Comment (melisabok) [reply venues/L4DC/2020/Conference/python/upload-submissions.py:70]: venueid can be the group id: 'L4DC.org/2020' or 'L4DC.org/2020/Conference'.

We should also add bibtex entry. What do you think?

Comment (celestemartinez) [reply venues/L4DC/2020/Conference/python/upload-submissions.py:70]: We need the paper forum for the bibtex url, so I have to first post the paper before adding the bibtex

**Comment by melisabok** [comment]
I think I can merge this, right?

**Comment by celestemartinez** [comment]
> I think I can merge this, right?

Yes please!

**Pull request merged**